<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSale</class>
 <widget class="QWidget" name="AddSale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="labelProduct">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Проданный товар</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="boxProduct">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="checkCustomer">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Заказчик</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="boxCustomer">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelPrice">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Цена за единицу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="spinPrice"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelAmount">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Колличество
проданного товара</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinAmount">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelDate">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Дата</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QDateEdit" name="date">
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2023</year>
             <month>1</month>
             <day>2</day>
            </datetime>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkCurrentDate">
           <property name="text">
            <string>Текущая дата</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="labelStatus">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save">
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
